src/main/webapp/script.js

```diff
@@ -11,54 +11,113 @@ function getDashboardItems() {
+        console.log('Failed to fetch movie from OMDb API: ' + error);
       });
 }
 
 /**
- * Creates a list element that displays information about
- * a specific entertainment item.
+ * Populates the item grid used in the Dashboard with all the entertainment
+ * items on the list.
+ *
+ * @param { jQuery } entertainmentItemsContainer - the div element used as a
+ *     parent in which to add the rows and columns of the grid
+ * @param { Array } entertainmentItemsList - the list of EntertainmentItems to
+ *     add to the grid
+ */
+function populateItemGrid(entertainmentItemsContainer, entertainmentItemsList) {
+  let currItemIndex = 0;
+
+  while (currItemIndex < entertainmentItemsList.length) {
+    const rowElem = $('<div class="row mb-4"></div>');
+
+    const MAX_COLS_PER_ROW = 3;
+
+    for (let col = 1; col <= MAX_COLS_PER_ROW &&
+         currItemIndex < entertainmentItemsList.length;
+         col++, currItemIndex++) {
+      const colElem = $('<div class="col-md-4"</div>');
+      colElem.append(
+          createEntertainmentItemCard(entertainmentItemsList[currItemIndex]));
+
+      rowElem.append(colElem);
+    }
+
+    entertainmentItemsContainer.append(rowElem);
+  }
+}
+
+/**
+ * Creates a card element that displays poster image, title, and description
+ * about a specific entertainment item.
+ *
+ * @param { JSON } entertainmentItem - the entertainment item whose
+ *     data will be displayed in the card element
+ * @returns { jQuery } card element representing the entertainment item
+ */
+function createEntertainmentItemCard(entertainmentItem) {
+  const card = $('<div class="card bg-light"></div>');
+  card.append(
+      $('<img class="card-img-top" src="' + entertainmentItem.imageUrl + '">'));
+  card.append(
+      $('<a class="stretched-link" href="item-page.html?itemId=' +
+        entertainmentItem.uniqueId + '"></a>'))
+
+  const cardBody = $('<div class="card-body"></div>');
+  cardBody.append(
+      $('<h5 class="card-title">' + entertainmentItem.title + '</h5>'));
+  cardBody.append(
+      $('<p class="card-text">' + entertainmentItem.description + '</p>'));
+
+  card.append(cardBody);
+
+  return card;
+}
+
+/**
+ * Creates a card element that displays poster image, title, and release date
+ * about an OMDbItem.
  *
- * @param { EntertainmentItem } entertainmentItem the entertainment item whose
- *     data will be displayed in the list element
- * @returns { jQuery } list element representing entertainment item
+ * @param { JSON } OMDbItem - item whose data will be displayed in the card element
+ * @returns { jQuery } card element representing the OMDbItem
  */
-function createEntertainmentItemListElem(entertainmentItem) {
-  const entertainmentItemElem = $('<li></li>');
-  const entertainmentItemLink =
-      $('<a href="item-page.html?itemId=' + entertainmentItem.uniqueId +
-        '"></a>');
+function createOMDbItemCard(OMDbItem) {
```

Comment:
`OMDbItem`  -> `omdbItem`

Reply:
OK I renamed the variables to omdbItem in last commit.